### PR TITLE
Added capability to pre-fill user name and e-mail on the feedback form

### PR DESCRIPTION
**Pull request by murki**

### CODE REVIEW

File: hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java [hunk 1]
```diff
@@ -663,7 +677,7 @@ public void run() {
     		SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
     		SimpleDateFormat formatNew = new SimpleDateFormat("d MMM h:mm a");
 
-    		Date date = null;
```
Comment (ranterle) [inline hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java]: Please fix indentation.

File: hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java [hunk 2]
```diff
@@ -328,11 +336,17 @@ protected void configureFeedbackView(boolean haveToken) {
           }
         }
         else {
-          /** We dont have Name and Email. Reset those fields */
-          nameInput.setText("");
-          emailInput.setText("");
+          /** We dont have Name and Email. Check if initial values  were provided */
```
Comment (ranterle) [inline hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java]: Please remove double whitespace in comment between "values" and "were".

File: hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java [hunk 3]
```diff
@@ -428,11 +442,11 @@ protected Dialog onCreateDialog(int id) {
     switch(id) {
       case DIALOG_ERROR_ID:
         return new AlertDialog.Builder(this)
-          .setMessage(getString(R.string.hockeyapp_dialog_error_message))
-          .setCancelable(false)
-          .setTitle(getString(R.string.hockeyapp_dialog_error_title))
-          .setIcon(android.R.drawable.ic_dialog_alert)
-          .setPositiveButton(getString(R.string.hockeyapp_dialog_positive_button), new DialogInterface.OnClickListener() {
+                .setMessage(getString(R.string.hockeyapp_dialog_error_message))
```
Comment (ranterle) [inline hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java]: Looks like no changes here - please remove indentation changes.

**Comment by ranterle** [comment]
Hi @murki, thanks for the PR - can you please change the commented lines and we'll happily merge it.

**Comment by murki** [comment]
Oops didn't notice my IDE changed these. Should be fixed now @ranterle

**Comment by murki** [comment]
Any updates on this PR @ranterle ?

**Comment by ranterle** [comment]
Sorry, my email client ate the notifications. Will merge now, thanks!

**Comment by murki** [comment]
Thanks @ranterle !

**Pull request merged**